test/powershell/Language/Scripting/Array.Tests.ps1

```diff
@@ -72,4 +72,36 @@ Describe "ArrayExpression Tests" -Tags "CI" {
         $result.Length | Should Be 1
         $result[0] | Should Be $null
     }
+
+    It "@([void](New-Item)) should create file" {
+        try {
+            $testFile = "$TestDrive\test.txt"
```

Comment:
I'd like to see more reliable tests - could we use the file name based on guid or do `$testFile | Should Not Exist`?

Reply:
Isn't the contract with Pester to create a new directory for every `It` block? Assuming that's the case, reliability shouldn't be a concern.

Reply:
When I see tons of files and directories in my temp folder I always wonder this though a debugger and Ctrl-C are the simplest ways to create trash. :-)
From docs:
- TestDrive is a PowerShell PSDrive for file activity limited to the scope of a single Describe or Context block.